### DISCUSSION TITLE
Send message

### DISCUSSION THREAD

**Discussion by Othales** [vasani-arpit/WBOT#282] [Q&A]
Is possible i send message, by PHP for number that i choice ?
type chat with person differents

example
![image](https://user-images.githubusercontent.com/24446510/142773915-eb2e23d9-0a0d-4e18-be8c-f7fd4bf94737.png)

**Comment by vasani-arpit** [comment]
It is possible to reply to the user via sends the message using Webhook. but sending a message is not possible without making significant change in code. 

you can reach out to me on email to discuss your requirement in detail.
